src/components/o-modal/_template.js

```diff
@@ -0,0 +1,23 @@
+import html from 'nanohtml';
+
+export default ({ open }, childrenFragment) => {
+  const closeModal = () => {
+    const modal = document.getElementById('modal-wrapper');
```

Comment:
Please do not use id.
Reason: There can be 2 modals on one page. Having both with the same ID will result in invalid HTML.